README.md

```diff
@@ -9,6 +9,38 @@ In Watchmen's `package.json` make sure to install [nightmareJS][nightmare].
 * [nightmare (@:octocat:)][nightmare]
 * [watchmen (@GH)](https://github.com/iloire/watchmen)
 
+### How To Use:
+- Create a nightmare script
+```
+var Nightmare = require('nightmare');
+
+nightmare
+  .goto('https://xfinity.com')
+  .type('#main-search-field', 'puppies')
+  .click('.search-button')
+  .evaluate(function() {
+    if (window.location.host === 'my.xfinity.com') {
+      return window.location.host;
+    }
+  })
+  .end()
+  .then(function(result) {
+    console.log('success', result);
+  })
+  .catch(function(err) {
+    console.log('error!', err);
+  });
+```
+
+- Copy file to a directory on your Watchmen instance
+- After uploaded - click add new service and select ping service `nightmare`
+- In Ping options `scriptPath` put full file path and file name
+- Click `Save` and you'll be on your way!
```

Comment:
could you add a screenshot that shows these things to click & set? want to make this incredibly simple to follow so people use it